doc/rho.1

```diff
@@ -1,25 +1,23 @@
-.TH rho 1 "August 10, 2017" "version 0.1" "rho User Commands"
+.TH rho 1 "August 29, 2017" "version 0.1" "rho User Commands"
```

Comment:
Should version state the same as `rho --version`?

Reply:
will update the version.